src/stepfunctions/steps/utils.py

```diff
@@ -45,3 +46,24 @@ def get_aws_partition():
             return cur_partition
 
     return cur_partition
+
+
+def merge_dicts(first, second, first_name, second_name):
+    """
```

Comment:
Please add some unit tests for this function

Reply:
Will add them with the next commit